content/en/docs/started/workstation/getting-started-multipass.md

```diff
@@ -24,37 +24,61 @@ Here's a summary of the steps involved:
 
 **Note:** the minimum version of Microk8s needed to enable Kubeflow is 1.18
 
-### 1. Install and set up Microk8s
+### 1. Install Microk8s
 
-Run the following commands to install and setup MicroK8s:
+Single-command install MicroK8s snap:
 
 ```
-snap install microk8s --classic
+sudo snap install microk8s --classic
+```
+Confirm install:
+```
 microk8s.status --wait-ready
-# Enable common services:
+```
+
+### 2. Enable Microk8s services:
+
+Enable common services:
```

Comment:
Use `- Enable...`. Again, I'd keep the dashes/hyphens to display there's a sequence of steps (1) Enable common services; 2) Enable GPU.). Check the Google style guide.